include/cell.tcc

```diff
@@ -856,6 +856,45 @@ Eigen::VectorXd mpm::Cell<Tdim>::compute_strain_rate(
   return strain_rate;
 }
 
+//! Compute strain rate for reduced integration at the centroid of cell
+template <unsigned Tdim>
+Eigen::VectorXd mpm::Cell<Tdim>::compute_strain_rate_centroid(unsigned phase) {
+  // Define strain rate
+  Eigen::VectorXd strain_rate;
```

Comment:
Since this function is called strain_rate_centroid, could you change this variable to `strain_rate_centroid`?